lib/osf-components/addon/components/osf-button/template.hbs

```diff
@@ -1 +1,3 @@
-{{yield}}
\ No newline at end of file
+<button local-class='OsfButton' ...attributes>
+    {{yield}}
+</button>
```

Comment:
@aaxelb @futa-ikeda What features need to be ported from `BsButton` to the new implementation `OsfButton`?